src/matchers/matcher/currentDate.matcher.js

```diff
@@ -0,0 +1,25 @@
+import Sugar from 'sugar-date';
+
+class CurrentDateMatcher {
+  isSatisfiedBy(prefix, name) {
+    return prefix === 'f' && name === 'currentDate';
```

Comment:
We're using prefix `f` for functions like `isVisible`, `isClickable` and so on. Maybe, in this case, it would be better to use a prefix `m`? What do you think @sethii ?

Reply:
Yeah actually might be better, this way

Reply:
Okay, let's keep it with `f:currentDate`